pkg/tlog/entry.go

```diff
@@ -248,6 +249,9 @@ func (entry *Entry) HasInclusionPromise() bool {
 func (entry *Entry) HasInclusionProof() bool {
 	return entry.logEntryAnon.Verification != nil
 }
+func (entry *Entry) Verification() *models.LogEntryAnonVerification {
```

Comment:
Is it necessary to export this function just for the tests? Could we instead construct the expected inclusion proof within the test?

Reply:
If I understand correctly, we will need the rekor private key from virtual sigstore to create the signedEntryTimestamp, so I will have to export that in virtual sigstore and copy the logic for creating SET and inclusion proof. 

Since `Verification()` exports the verification material for the tlog entry, I thought it would be okay to export it

Reply:
I think this can be avoided, more in tlog_test.go below.